src/snakemake/report/__init__.py

```
@@ -541,21 +547,34 @@ def get_time(rectime, metatime, sel_func):
+
+                    async def expand_cat_name(cat_name):
+                        if cat_name is not None:
+                            return await expand_report_argument(
+                                cat_name, wildcards, job
+                            )
+                        else:
+                            return cat_name
+
                     category = Category(
-                        name=report_obj.category, wildcards=wildcards, job=job
+                        name=await expand_cat_name(report_obj.category),
```

Ruff (0.8.2)

Function definition does not bind loop variable `report_obj`

(B023)